spec/fixtures/test_config_mixin.rb

```diff
@@ -0,0 +1,6 @@
+require 'shanty/config_mixin'
```

Comment:
I did that because rubocop complains about the include constant being the same as the described class